node/core/dispute-coordinator/src/scraping/candidates.rs

```diff
@@ -17,8 +25,16 @@ impl RefCountedCandidates {
 	// If `CandidateHash` doesn't exist in the `HashMap` it is created and its reference
 	// count is set to 1.
 	// If `CandidateHash` already exists in the `HashMap` its reference count is increased.
-	pub fn insert(&mut self, candidate: CandidateHash) {
-		*self.candidates.entry(candidate).or_default() += 1;
+	pub fn insert(
+		&mut self,
+		block_number: BlockNumber,
+		block_hash: Hash,
+		candidate: CandidateHash,
```

Comment:
nit: I expected the key of the map to be the first argument, and not the last. It is a bit jarring this way, as it's atypical for an `insert` method.